packages/riipen-ui/src/components/VerticalProgressBarItem.jsx

```diff
@@ -30,35 +29,29 @@ const VerticalProgressBarItem = ({ progress }) => {
 
   const linkedStyles = getLinkedStyles();
 
-  const iconClasses = [
-    "iconWrapper",
-    progress.color && `${progress.color}`
-  ].filter(Boolean);
-
-  const className = clsx(
-    "progress",
-    progress.color && !progress.noBar && `progress-${progress.color}`,
-    progress.noBar && "progress-none"
-  );
-
   return (
-    <React.Fragment>
```

Comment:
Remove wrapper React.Fragment (not necessary)

Reply:
Code I think I wrote in my first couple months at Riipen 🙈 

At least you wrote the actual component on web first, it was wizardry to me at the time.